### PR TITLE
save joint limits of space in MetaSkeletonStateSpaceSaver

### PR DESCRIPTION
**Pull request by dqyi11**
resolving personalrobotics/aikido #218

### CODE REVIEW

File: include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp [hunk 1]
```diff
@@ -16,7 +18,20 @@ MetaSkeletonStateSpaceSaver::~MetaSkeletonStateSpaceSaver()
     std::cerr << "[MetaSkeletonStateSpaceSaver] The number of DOFs in the "
               << "MetaSkeleton does not match the saved state.";
```
Comment (brianhou) [inline include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp]: `saved state` -> `saved position`.

File: include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp [hunk 2]
```diff
@@ -16,7 +18,20 @@ MetaSkeletonStateSpaceSaver::~MetaSkeletonStateSpaceSaver()
...
   }
+  if (mPositionLowerLimits.size() != mSpace->getMetaSkeleton()->getNumDofs())
+  {
+    std::cerr << "[MetaSkeletonStateSpaceSaver] The number of DOFs in the "
+              << "MetaSkeleton does not match the save joint lower limits.";
```
Comment (brianhou) [inline include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp]: `save joint lower limits` -> `saved position lower limits`.

File: include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp [hunk 3]
```diff
@@ -16,7 +18,20 @@ MetaSkeletonStateSpaceSaver::~MetaSkeletonStateSpaceSaver()
...
+  }
+  if (mPositionUpperLimits.size() != mSpace->getMetaSkeleton()->getNumDofs())
+  {
+    std::cerr << "[MetaSkeletonStateSpaceSaver] The number of DOFs in the "
+              << "MetaSkeleton does not match the save joint upper limits.";
```
Comment (brianhou) [inline include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp]: `save joint upper limits` -> `saved position upper limits`.

File: include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp (hunk 1 above)
Comment (dqyi11) [reply include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp]: updated

File: include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp (hunk 2 above)
Comment (dqyi11) [reply include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp]: Updated

File: include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp (hunk 3 above)
Comment (dqyi11) [reply include/aikido/statespace/dart/detail/MetaSkeletonStateSpaceSaver-impl.hpp]: Updated

**Pull request merged**